apis/v1beta1/shared_types.go

```diff
@@ -275,6 +275,15 @@ const (
 	// This reason is used with the "ResolvedRefs" condition when one of the
 	// Route's rules has a reference to a resource that does not exist.
 	RouteReasonBackendNotFound RouteConditionReason = "BackendNotFound"
+
+	// This condition indicates whether incompatible filters are present on
+	// a Route. Controllers should only add this condition with status true
+	// when an incompatible combination is present.
+	RouteConditionIncompatibleFilters RouteConditionType = "IncompatibleFilters"
+
+	// This reason is used with the "IncompatibleFilters" condition when the
+	// condition is true.
+	RouteReasonIncompatibleFilters RouteConditionReason = "IncompatibleFilters"
```

Comment:
Should this just be another reason for the "Accepted" condition that is defined beside UnsupportedValue? I'm not sure we need a new condition for this, but I could be missing some context.

Reply:
Does this perhaps imply adding a `Conflicted` RouteConditionType similar to https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1beta1.ListenerConditionType to properly communicate the partial acceptance case?

Reply:
This reason is now the only initial option available on the new "PartiallyInvalid" condition

Reply:
@sunjayBhatia thanks for the reference to the text of that GEP! It seems like what that's saying is:

1) When this happens and there's still at least one valid Route rule, accept the valid route rules but don't program the invalid one.
2) When this happens and there are no valid Route rules, don't accept the Route at all.

In that case it seems like there would be a subset of reasons that could be added to `Accepted` in 2) or `PartiallyInvalid` in 1). For example the following reasons seem like they would also be equally valid for both conditions (in addition to the proposed `IncompatibleFilters` reason).

* RefNotPermitted
* UnsupportedValue

/cc @youngnick

Reply:
should be resolved from previous commits, so resolving this thread